xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/liveDataSource.js

```diff
@@ -77,8 +106,33 @@ define('xwiki-livedata-source', ['module', 'jquery'], function(module, $) {
     return Promise.resolve($.post(getEntriesURL(source), entry).then(entry => entry.values));
   };
 
+  const updateEntry = function(source, entryId, entry) {
+    return Promise.resolve($.post({
+      url: getEntryPropertiesURL(source, entryId),
+      contentType: 'application/json',
+      data: JSON.stringify({
+        values: entry
+      })
+    }));
+  }
```

Comment:
not sure if it is best to only update the edited entry from the response of the request, or to simply reload LD.
The second solution is easier and help keeping the displayed data in sync with the server, but might move the edited entry out of view of the user with some combinations of filters and sort.